modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/spi/display/PermissionTemplateCTDisplayRenderer.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.change.tracking.web.internal.spi.display;
+
+import com.liferay.change.tracking.spi.display.BaseCTDisplayRenderer;
+import com.liferay.change.tracking.spi.display.CTDisplayRenderer;
+import com.liferay.permission.template.model.PermissionTemplate;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.model.ClassName;
+import com.liferay.portal.kernel.security.permission.ResourceActions;
+import com.liferay.portal.kernel.service.ClassNameLocalService;
+
+import java.util.Locale;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Quan Huynh
+ */
+@Component(service = CTDisplayRenderer.class)
+public class PermissionTemplateCTDisplayRenderer
+	extends BaseCTDisplayRenderer<PermissionTemplate> {
+
+	@Override
+	public Class<PermissionTemplate> getModelClass() {
+		return PermissionTemplate.class;
+	}
+
+	@Override
+	public String getTitle(
+		Locale locale, PermissionTemplate permissionTemplateSetting) {
+
+		ClassName className = _classNameLocalService.fetchClassName(
+			permissionTemplateSetting.getClassNameId());
+
+		if (className == null) {
+			return null;
+		}
+
+		return _language.format(
+			locale, "x-for-x",
+			new String[] {
+				_resourceActions.getModelResourceNamePrefix() +
+					PermissionTemplate.class.getName(),
+				_resourceActions.getModelResource(
+					locale, className.getClassName())
+			});
+	}
+
+	@Override
+	public boolean isHideable(PermissionTemplate permissionTemplateSetting) {
```

Comment:
permissionTemplate instead of permissionTemplateSetting?

Reply:
Fixed it. Thanks